### ISSUE TITLE
Update Language for Server-Side Validation

### ISSUE THREAD

**Issue by dgp1130** [uniform-team/Uniform-Validation-Language#20]
The language is currently not able to be validated on the server due to a hidden dependency on the HTML. Anytime the jQuery operator is used `$(...)`, the client references the HTML in a way that the server cannot effectively replicate. The language will need to change to be validatable on the server. I suggest changing jQuery selectors to refer to elements by the `name` attribute. This will give the client and server some common ground on what the core data is.

Instead of:
```
$("#myValueId") {
    valid: ...;
}
```

It should be:
```
myValueName {
    valid: ...;
}
```

Removing jQuery means that there will be no way to refer to multiple selectors, so I suggest making an array syntax within the actual Uniform code.

Instead of:
```
$(".myValuesClass") {
    // Rule applies to every element in array
    valid: this matches /"."/;
}

@myValuesValid: all $(".myValuesClass") is valid;
```

It should be:
```
@myValues: [ myValueName1, myValueName2 ];

@myValues {
    // Rule applies to every element in array
    valid: this matches /"."/;
}

@myValuesValid: all @myValues is valid; // Existing array syntax can still be used
```

This allows the server to know from the Uniform file what each array is and how validity is defined for it, rather than leaving that information hidden in an unseen HTML class.

Other changes may be necessary as well.

**Comment by sawyernovak** [comment]
Is this a duplicate/ already completed?

**Comment by sawyernovak** [comment]
Fixed by #31
